lib/topologies/server.js

```diff
@@ -389,6 +389,13 @@ Object.defineProperty(Server.prototype, 'port', {
   }
 });
 
+Object.defineProperty(Server.prototype, 'autoEncrypter', {
+  enumerable: true,
+  get: function() {
+    return this.s.options.autoEncrypter;
```

Comment:
We need to do the check from the original `isClientEncryptionEnabled` method here in case `server.s` is not defined: `if (server.s && server.s.options && server.s.options.autoEncrypter)`